redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/actions/CertReplaceInRedBeamsAction.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.redbeams.flow.redbeams.start.actions;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.redbeams.api.model.common.DetailedDBStackStatus;
+import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsContext;
+import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsEvent;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertReplaceInRedBeamsRequest;
+import com.sequenceiq.redbeams.service.stack.DBStackStatusUpdater;
+
+@Component("CERT_REPLACE_STATE")
+public class CertReplaceInRedBeamsAction extends AbstractRedbeamsStartAction<RedbeamsEvent> {
```

Comment:
Possibly rename to `CertRotateInRedbeamsAction`. (I think `Beams` need not be capitalized, and `Rotate` expresses the purpose more properly.)